execution/execution_test.go

```diff
@@ -5,213 +5,55 @@ import (
 	"testing"
 	"testing/quick"
 
-	"github.com/mesg-foundation/core/service"
-	"github.com/mesg-foundation/core/x/xstructhash"
 	"github.com/stretchr/testify/require"
 )
 
-var (
```

Comment:
@krhubert could you move those test in sdk package in a new PR?
The code coverage of sdk is really low: https://codecov.io/gh/mesg-foundation/engine/compare/9cf2d0b48817a45110ca0189e74419b232f137d0...f20bddbc143aa30752d4d07be3be95b76060e68a/src/sdk/sdk.go